src/MissionCreator.sol

```diff
@@ -121,47 +162,78 @@ contract MissionCreator is Ownable, IERC721Receiver {
                 dataHook: address(launchPadPayHook),
                 metadata: 0 // No metadata is attached to this ruleset.
             }),
-            splitGroups: new JBSplitGroup[](2), // Initialize as dynamic array
+            splitGroups: splitGroups, // Initialize as dynamic array
             fundAccessLimitGroups: new JBFundAccessLimitGroup[](1) // Initialize as dynamic array
         });
+
+        JBCurrencyAmount[] memory surplusAllowances = new JBCurrencyAmount[](1);
+        surplusAllowances[0] = JBCurrencyAmount({
+            amount: uint224(128_000_000 * 10 ** 18), // 128 million ETH
```

Comment:
Just curious, why this number?

Reply:
just wanted to leave it functionally unlimited. picked a round number which is larger than the current ETH supply. just in case.

added a comment.

Reply:
Could go for MAX_INT or something - although uint224 is pretty weird. Sounds fine to me, unless in far off, dystopian future where ETH hyperinflates